src/components/PokemonDisplay.js

```diff
@@ -0,0 +1,107 @@
+import React, {setState, useEffect} from 'react';
+import $ from 'jquery'
+import Button from '@material-ui/core/Button'
+
+class PokemonDisplay extends React.Component {
+
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            index: 0,
+            value: "",
+            pokemon: {
+                name:				"",
+                nationalPokedexNum: -1,
+                artworkUrl:			"",
+                type:				"",
+                species:			"",
+                height:				"",
+                weight:				""
+            }
+        }
+        this.CreatePokemonObject = this.CreatePokemonObject.bind(this);
+        this.GetWebsiteHTML = this.GetWebsiteHTML.bind(this);
+        this.StartScrape = this.StartScrape.bind(this);
+        this.componentDidUpdate = this.componentDidUpdate.bind(this);
+        this.componentDidMount = this.componentDidMount.bind(this);
+    }
+
+    async GetWebsiteHTML(webUrl)
+     {
+        var result = await $.ajax({ 
```

Comment:
Worth investigating if the electronjs framework has classes for web requests, and consider using those instead of ajax.  Likewise when going to potentially save local copy of dictionary to a file look to use electron file system classes.